queries/highlights.scm

```diff
@@ -1,31 +1,29 @@
-(identifier) @variable
```

Comment:
Why remove this one? Is it conflicting with something?

Reply:
For helix at least, that would match every identifier as a variable, regardless of the rest of the file due to how it parses in comparison to neovim

Reply:
I guess we'd want more sophisticated matching of local variables anyways: https://tree-sitter.github.io/tree-sitter/syntax-highlighting#local-variables

Reply:
I have the start of a `locals.scm` above that uses those actually

Reply:
Yep, I noticed it was matching too much due to being too generic.

So I removed in a attempt to avoid multiple matches for the same node.